executor/simple.go

```diff
@@ -1031,6 +1382,25 @@ func (e *SimpleExec) executeAlterUser(ctx context.Context, s *ast.AlterUserStmt)
 		}
 	}
 
+	sysSession, err := e.getSysSession()
+	defer clearSysSession(ctx, sysSession)
+	if err != nil {
+		return err
+	}
+	sqlExecutor := sysSession.(sqlexec.SQLExecutor)
+	// session isolation level changed to READ-COMMITTED
+	_, err = sqlExecutor.ExecuteInternal(ctx, "set tx_isolation = 'READ-COMMITTED'")
+	if err != nil {
+		return err
+	}
+	if _, err := sqlExecutor.ExecuteInternal(ctx, "BEGIN PESSIMISTIC"); err != nil {
+		_, errRollback := sqlExecutor.ExecuteInternal(ctx, "rollback")
```

Comment:
Since the start of explicit transaction is failed, we don't need an explicit rollback.

(Also, the rollback will be executed in `clearSysSession()`.

Reply:
done